public/controllers/management/components/management/configuration/configuration-switch.js

```diff
@@ -99,12 +102,8 @@ class WzConfigurationSwitch extends Component {
   async componentDidMount() {
     // If agent, check if is synchronized or not
     if (this.props.agent.id !== '000') {
-      try {
-        const agentSynchronized = await agentIsSynchronized(this.props.agent);
-        this.setState({ agentSynchronized });
-      } catch (error) {
-        // do nothing
-      }
+      const agentSynchronized = await agentIsSynchronized(this.props.agent);
```

Comment:
Why you have deleted this try catch block?

Reply:
You're right. I revert this change.